goesvfi/integrity_check/gui_tab.py:358: error: Call to untyped function "MissingTimestampsModel" in typed context  [no-untyped-call]
goesvfi/integrity_check/remote/s3_store.py:857: error: Call to untyped function "log_error" in typed context  [no-untyped-call]
goesvfi/integrity_check/render/netcdf.py:119: error: Call to untyped function "masked_less_equal" in typed context  [no-untyped-call]
goesvfi/integrity_check/goes_imagery.py:511: error: Call to untyped function "GOESImageryDownloader" in typed context  [no-untyped-call]
goesvfi/integrity_check/goes_imagery.py:517: error: Call to untyped function "GOESImageProcessor" in typed context  [no-untyped-call]
goesvfi/integrity_check/goes_imagery.py:610: error: Call to untyped function "GOESImageryManager" in typed context  [no-untyped-call]
goesvfi/integrity_check/visualization_manager.py:211: error: Call to untyped function "get_band_colormap" in typed context  [no-untyped-call]
goesvfi/integrity_check/visualization_manager.py:291: error: Call to untyped function "get_time_directory" in typed context  [no-untyped-call]
goesvfi/integrity_check/visualization_manager.py:294: error: Call to untyped function "get_filename" in typed context  [no-untyped-call]
goesvfi/integrity_check/visualization_manager.py:370: error: Call to untyped function "list_available_visualizations" in typed context  [no-untyped-call]
goesvfi/integrity_check/visualization_manager.py:402: error: Call to untyped function "process_band_image" in typed context  [no-untyped-call]
goesvfi/integrity_check/visualization_manager.py:406: error: Call to untyped function "process_band_image" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:46: error: Call to untyped function "VisualizationManager" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:121: error: Call to untyped function "_search_day_hours" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:129: error: Call to untyped function "_search_day_hours" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:151: error: Call to untyped function "_search_day_hours" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:165: error: Call to untyped function "_search_day_hours" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:332: error: Call to untyped function "_try_all_web_sources" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:347: error: Call to untyped function "_try_noaa_cdn" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:354: error: Call to untyped function "_try_noaa_cdn" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:361: error: Call to untyped function "_try_noaa_cdn" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:367: error: Call to untyped function "_try_rammb_slider" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:373: error: Call to untyped function "_try_archived_imagery" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:449: error: Call to untyped function "_download_image" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:493: error: Call to untyped function "_download_image" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:532: error: Call to untyped function "_download_image" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:629: error: Call to untyped function "get_display_name" of "ExtendedChannelType" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:638: error: Call to untyped function "download_sample_data" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:642: error: Call to untyped function "process_sample_netcdf" in typed context  [no-untyped-call]
goesvfi/integrity_check/sample_processor.py:656: error: Call to untyped function "download_web_sample" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:111: error: Call to untyped function "get_estimated_processing_time" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:240: error: Call to untyped function "get_display_name" of "ExtendedChannelType" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:950: error: Call to untyped function "VisualizationManager" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:956: error: Call to untyped function "SampleProcessor" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:962: error: Call to untyped function "GOESImageryManager" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:980: error: Call to untyped function "get_display_name" of "ExtendedChannelType" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:994: error: Call to untyped function "get_estimated_processing_time" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:1007: error: Call to untyped function "get_display_name" of "ExtendedChannelType" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_imagery_tab.py:1078: error: Call to untyped function "get_display_name" of "ExtendedChannelType" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_gui_tab.py:51: error: Call to untyped function "__init__" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_gui_tab.py:1227: error: Call to untyped function "EnhancedMissingTimestampsModel" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_gui_tab.py:1634: error: Call to untyped function "AdvancedOptionsDialog" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_gui_tab.py:1670: error: Call to untyped function "BatchOperationsDialog" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_gui_tab.py:2507: error: Call to untyped function "CDNConfigDialog" in typed context  [no-untyped-call]
goesvfi/integrity_check/enhanced_gui_tab.py:2517: error: Call to untyped function "AWSConfigDialog" in typed context  [no-untyped-call]
goesvfi/integrity_check/combined_tab.py:101: error: Call to untyped function "_switch_tab" in typed context  [no-untyped-call]
goesvfi/integrity_check/combined_tab.py:102: error: Call to untyped function "_switch_tab" in typed context  [no-untyped-call]
